### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by iTitus**

### CODE REVIEW

File: src/main/java/buildcraftAdditions/config/ConfigurationHandler.java [hunk 1]
```diff
@@ -216,25 +236,8 @@ public static void readConfig() {
 		eurekaIntegration = configFile.get("Misc", "eurekaIntegration", true).setRequiresMcRestart(true).getBoolean();
 		dusterParticles = configFile.get("Misc", "dusterParticles", true).getBoolean();
 		particleCount = configFile.get("Misc", "particleCount", 100).setMinValue(0).getInt();
-		forceEnableBCRefinery = !enabled("MultiBlockRefining") || configFile.get("Misc", "forceEnableBCRefinery", false).setRequiresMcRestart(true).getBoolean();
+		forceEnableBCRefinery = !enabled("MultiBlockRefining") | configFile.get("Misc", "forceEnableBCRefinery", false).setRequiresMcRestart(true).getBoolean();
```
Comment (AEnterprise) [inline src/main/java/buildcraftAdditions/config/ConfigurationHandler.java]: why this change?

Comment (iTitus) [inline src/main/java/buildcraftAdditions/config/ConfigurationHandler.java]: To force it to add the config option.
